config.yml.default

```diff
@@ -157,32 +157,33 @@ greeting:
   goodbye_spectators: "Thanks for watching!" # Message to send to spectator chat at the end of a game
 
 # pgn_directory: "game_records" # A directory where PGN-format records of the bot's games are kept
 
 matchmaking:
-  allow_matchmaking: false    # Set it to 'true' to challenge other bots.
-  challenge_variant: "random" # If set to 'random', the bot will choose one variant from the variants enabled in 'challenge.variants'.
-  challenge_timeout: 30       # Create a challenge after being idle for 'challenge_timeout' minutes. The minimum is 1 minute.
-  challenge_initial_time:     # Initial time in seconds of the challenge (to be chosen at random).
+  allow_matchmaking: true          # Set it to 'true' to challenge other bots.
+  challenge_variant: "random"      # If set to 'random', the bot will choose one variant from the variants enabled in 'challenge.variants'.
+  challenge_timeout: 30            # Create a challenge after being idle for 'challenge_timeout' minutes. The minimum is 1 minute.
+  challenge_initial_time:          # Initial time in seconds of the challenge (to be chosen at random).
     - 60
-    - 120
-  challenge_increment:        # Increment in seconds of the challenge (to be chosen at random).
+    - 180
+  challenge_increment:             # Increment in seconds of the challenge (to be chosen at random).
     - 1
     - 2
-#  challenge_days:            # Days for correspondence challenge (to be chosen at random).
+    - 0
+#  challenge_days:                 # Days for correspondence challenge (to be chosen at random).
 #    - 1
 #    - 2
-# opponent_min_rating: 600    # Opponents rating should be above this value (600 is the minimum rating in lichess).
-# opponent_max_rating: 4000   # Opponents rating should be below this value (4000 is the maximum rating in lichess).
-  opponent_rating_difference: 100 # The maximum difference in rating between the bot's rating and opponent's rating.
+# opponent_min_rating: 600         # Opponents rating should be above this value (600 is the minimum rating in lichess).
+# opponent_max_rating: 4000        # Opponents rating should be below this value (4000 is the maximum rating in lichess).
+  opponent_rating_difference: 300  # The maximum difference in rating between the bot's rating and opponent's rating.
   opponent_allow_tos_violation: true # Set to 'false' to prevent challenging bots that violated Lichess Terms of Service.
-  challenge_mode: "random"    # Set it to the mode in which challenges are sent. Possible options are 'casual', 'rated' and 'random'.
-  challenge_filter: none  # If a bot declines a challenge, do not issue a similar challenge to that bot. Possible options are 'none', 'coarse', and 'fine'.
-# block_list:                 # The list of bots that will not be challenged
+  challenge_mode: "standard"       # Set it to the mode in which challenges are sent. Possible options are 'casual', 'rated' and 'random'.
```

Comment:
`standard` isn't an option. 'casual', 'rated' and 'random' are the possible options and I prefer `random`.

Reply:
Okay, changed to `random`